src/View/Teradata/TeradataViewReflection.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\View\Teradata;
+
+use Doctrine\DBAL\Connection;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Keboola\TableBackendUtils\View\ViewReflectionInterface;
+
+class TeradataViewReflection implements ViewReflectionInterface
+{
+    private Connection $connection;
+
+    private string $databaseName;
+
+    private string $viewName;
+
+    public function __construct(Connection $connection, string $databaseName, string $viewName)
+    {
+        $this->databaseName = $databaseName;
+        $this->viewName = $viewName;
+        $this->connection = $connection;
+    }
+
+    public function getDependentViews(): array
+    {
+        $views = $this->connection->fetchAllAssociative(
+            sprintf(
+                'SELECT * FROM dbc.tables WHERE tablekind=%s AND databasename=%s',
+                TeradataQuote::quote('V'),
+                TeradataQuote::quote($this->databaseName)
+            )
+        );
+
+        /**
+         * @var array{
+         *  database_name: string,
```

Comment:
@zajca tu by sme mali nieco vymysliet pretoze teradata nema schemu ale ten interface to vyzaduje :/

Reply:
Hmm takze pre teraz to mam volat `schema_name` a pak nieco vymyslime jo ?